src/code/code_800EC960.c

```diff
@@ -3931,22 +3933,23 @@ void func_800F4010(Vec3f* pos, u16 sfxId, f32 arg2) {
             sfxId2 = NA_SE_PL_METALEFFECT_KID;
         }
         D_8016B7AC = (sp24 * 0.7) + 0.3;
-        Audio_PlaySoundGeneral(sfxId2, pos, 4, &D_8016B7B0, &D_8016B7AC, &D_801333E8);
+        Audio_PlaySfxGeneral(sfxId2, pos, 4, &D_8016B7B0, &D_8016B7AC, &gSfxDefaultReverb);
     }
 }
 
 void func_800F4138(Vec3f* pos, u16 sfxId, f32 arg2) {
     func_800F3F84(arg2);
-    Audio_PlaySoundGeneral(sfxId, pos, 4, &D_8016B7B0, &D_8016B7A8, &D_801333E8);
+    Audio_PlaySfxGeneral(sfxId, pos, 4, &D_8016B7B0, &D_8016B7A8, &gSfxDefaultReverb);
 }
 
 void func_800F4190(Vec3f* pos, u16 sfxId) {
-    Audio_PlaySoundGeneral(sfxId, pos, 4, &D_801305B0, &D_801333E0, &D_801305B4);
+    Audio_PlaySfxGeneral(sfxId, pos, 4, &D_801305B0, &gSfxDefaultFreqAndVolScale, &D_801305B4);
 }
 void Audio_PlaySoundRandom(Vec3f* pos, u16 baseSfxId, u8 randLim) {
```

Comment:
Is the plan to rename the other PlaySound functions as well, like this one?

I feel like they should all be changed at once ideally, and if we do it over time I'm a bit worried we'll forget some functions like `Audio_PlayActorSound2` or the less used variants, but that might just be me

Reply:
Exactly, the intent is to propose changing `Audio_PlaySoundXXX` to `Audio_PlaySfxXXX`. I wanted to change `Audio_PlaySoundGeneral` on it's own since it's used so often, but yes the `Sound` -> `Sfx` could use a discussion. Which if you feel that needs more discussion, I could always revert back to `Audio_PlaySoundGeneral` so it doesn't hold up this PR.

I also have my eyes on all those collection of audio sfx functions from z_actor, z_lib, and other wrappers through different files. I've been itching to change `Audio_PlayActorSound2` for a while since a function outside the audio library probably should have the `Audio_` prefix? I've already brainstormed ideas for all of them, it's just a matter of getting to it. I don't want too many audio PR's open at once

Reply:
Alright, thank you for the explanation.

So yeah right now I think it would be best to revert to `Audio_PlaySoundGeneral` for this PR, and discuss the `Sound` -> `Sfx` change in a PR dedicated to that, as well as any renames for the other wrappers like you mentioned